modules/virt-deprecated-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-deprecated-tasks_{context}"]
+= Removing deprecated or unused resources
+
+* The tekton-tasks-operator (TTO) has been deprecated and removed in {VirtProductName} 4.17. You can clean up any remaining resources associated with the operator and its pipelines by running the following command.
```

Comment:
maybe don't mention pipelines explicitly.. 
resources associated with the operator should cover all